services/motion/builtin/builtin.go

```diff
@@ -276,22 +276,44 @@ func (ms *builtIn) MoveOnGlobe(
 	if err != nil {
 		return false, err
 	}
+	currentPosition := currentPIF.Pose().Point()
+
+	// get position of localizer relative to base
+	robotFS, err := ms.fsService.FrameSystem(ctx, nil)
+	if err != nil {
+		return false, err
+	}
+
+	// build maps of relevant components and inputs from initial inputs
+	fsInputs, _, err := ms.fsService.CurrentInputs(ctx)
```

Comment:
Should move this inside check of `localizerFrame != nil` since it's unnecessary to get otherwise